config.yml

```diff
@@ -1,5 +1,5 @@
 # Data Locations
-aia_path: /data/sw/version3/
+aia_path: /home/solardynamo/network_data
```

Comment:
It is best to not change these config paths in the commits you want merged in. You can change it locally and never commit it.